proposal/xyhuang7215/proposal.rst

```diff
@@ -0,0 +1,52 @@
+===========================
+Proposal for SekirOgre
+===========================
+
+
+Basic information
+=================
+
+Recently, I enjoy playing Sekiro: Shadows Die Twice. So I think it might be
+a good and interesting practice to build my own game engine specially for this
+kind of action adventure game.
+
+Since it seems hard to make a good renderning engine from scratch, I decide to
+use ogre engine (Object-Oriented Graphics Rendering Engine) , a 3D rendering
+engine written in C++. So I can just focus on building other components of
+engine, like physics and collision.
```

Comment:
You should make it explicit for what you want to do in the project.  "Other components like" is not a good way to specify.